### PR TITLE
ANDROID-14543 Checkbox text alignment

### PR DESCRIPTION
**Pull request by pmartinbTEF**
### :goal_net: What's the goal?

CheckBoxInput component has its text misaligned in XML implementation

### :construction: How do we do it?
Added small padding to the text
Updated screenshots
Also configure Lint to not mark library updates as an error

### ☑️ Checks
- [x] I updated the documentation, including readmes and wikis. If this is a breaking change, update [UPGRADING.md](../UPGRADING.md) to inform users how to proceed. If no updates are necessary, indicate so.
- [x] Tested with dark mode.
- [x] Tested with API 21.

### :test_tube: How can I test this?
- [x] Execute Mística catalog app
- [x] Go to inputs section
- [x] Compare Checkbox from Classic and Compose versions

Before:
drawing

After:
drawing

### CODE REVIEW

File: library/src/main/res/layout/input_checkbox.xml [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-            android:checked="true"
-        android:layout_height="wrap_content"
-        tools:text="@tools:sample/lorem/random" />
+	<androidx.appcompat.widget.AppCompatCheckBox
+			android:id="@+id/checkbox_input"
+			style="@style/AppTheme.Forms.CheckBox"
+			android:layout_width="match_parent"
+			android:checked="true"
+			android:layout_height="wrap_content"
+			android:paddingTop="3dp"
```
Comment (pmartinbTEF) [inline library/src/main/res/layout/input_checkbox.xml:15]: This is the only change, the rest is only format

File: app/lint.xml [hunk 2]
```diff
@@ -7,4 +7,5 @@
     <issue id="NotificationPermission">
         <ignore regexp="com.microsoft.appcenter.distribute.Distribute" />
     </issue>
+    <issue id="GradleDependency" severity="ignore" />
```
Comment (pmartinbTEF) [inline app/lint.xml:10]: As configured in other projects not mark library updates as errors

**Review by jeprubio — APPROVED** [review]
Great 👏

**Review by haynlo — APPROVED** [review]
👍🏼

**Review by aweell — APPROVED** [review]

**Pull request merged**